Jesusborjas006/TrackMyBite
# Feat/users profile

### What is the feature?
Main feature is to create a an user profile page to view user details such as calorie goal, age, weight, etc. User is able to make changes to their profile by going to the edit page. Added an extra input for calorie goal in the login page so that user's can set their own goal and be displayed in the home page.

### What is the solution?
Create two more routes for profile and edit page and pass userInfo state to both of these components. Added more key value pairs to the userInfo state so that more information can be stored in a single spot.

### What areas of the site does it impact?
This impacts the login, home, profile, and edit page. Styling had been modified across application.

### How to test?
Cypress can be used to test if the users are able to see the new displayed data in the profile page. User's can make changes in the edit and see if it updates in the home and profile card.

## Other Notes
closes #1 
closes #3 
closes #18
